sources/packages/backend/apps/api/src/app.system-access.module.ts

```diff
@@ -66,19 +63,22 @@ import { WorkflowClientService, SequenceControlService } from "@sims/services";
 // todo: once all integration are moved. remove sshservice.
 import { SshService } from "@sims/integrations/services";
 import { IER12IntegrationModule } from "@sims/integrations/institution-integration/ier12-integration/ier12-integration.module";
+import { QueueService } from "@sims/services/queue";
+import { TypeOrmModule } from "@nestjs/typeorm";
 
 @Module({
   imports: [
     DatabaseModule,
     AuthModule,
     SINValidationModule,
     IER12IntegrationModule,
+    TypeOrmModule.forFeature(DBEntities),
```

Comment:
Instead of repeating it everywhere, can we try the below? I guess that it is possible we just did not try before 😉 
![image](https://user-images.githubusercontent.com/61259237/207122294-26e10210-4db0-4ab9-8779-8bbf9c2ef11f.png)